site/cds_rdm/inspire_harvester/reader.py

```diff
@@ -61,32 +73,51 @@ def _iter(self, url, *args, **kwargs):
 
     def read(self, item=None, *args, **kwargs):
         """Builds a query depending on the input data."""
+        current_app.logger.info("Start reading data from INSPIRE.")
+
         document_type = "thesis"
         oai_set = "ForCDS"
 
         if self._inspire_id:
             # get by INSPIRE id
+            current_app.logger.debug(
+                "INSPIRE ID provided. Building query params for request to INSPIRE based on that."
+            )
```

Comment:
```suggestion
            current_app.logger.info(
                f"Reading record ID {self._inspire_id} from INSPIRE."
            )
```
I think this might be useful in info level